streams/src/test/java/org/apache/kafka/streams/integration/QueryableStateIntegrationTest.java

```diff
@@ -143,6 +143,7 @@ public void before() throws IOException {
         streamsConfiguration
             .put(StreamsConfig.VALUE_SERDE_CLASS_CONFIG, Serdes.String().getClass());
         streamsConfiguration.put(StreamsConfig.CACHE_MAX_BYTES_BUFFERING_CONFIG, cacheSizeBytes);
+        streamsConfiguration.put(StreamsConfig.COMMIT_INTERVAL_MS_CONFIG, 1000);
```

Comment:
Strictly speaking, no, but...
One of the tests was timing out occasionally (30 second timeout) and this was happening because the default commit interval is 30 seconds. When caching is enabled there will be no flushing of state for 30 seconds, so the test either takes at least that long to complete successfully or it takes that long and fails.
So this means the test is much more reliable and completes within a few seconds.